versions/development/SPEC.md

```diff
@@ -698,6 +699,48 @@ Pair values can also be specified within the [workflow inputs JSON](https://gith
 }
 ```
 
+### Optional literals
+
+Any non-optional value can be wrapped into an optional value by assigning it to an optional variable:
```

Comment:
wrapped -> turned into? 

I think that wrapped is too programmer-specific, not english.

Reply:
updated

Reply:
Wrapping seems imprecise anyway since this is an implicit type conversion, e.g. "any expression of a non-optional type can be implicitly convert to an optional version of that type."